[2018-02-21 17:59:02,254] {cli.py:374} INFO - Running on host Sanjeevs-MacBook-Pro.local
[2018-02-21 17:59:02,272] {models.py:1197} INFO - Dependencies all met for <TaskInstance: property_data.run_new_listings_from_remax 2018-02-19 02:10:00 [queued]>
[2018-02-21 17:59:02,275] {models.py:1197} INFO - Dependencies all met for <TaskInstance: property_data.run_new_listings_from_remax 2018-02-19 02:10:00 [queued]>
[2018-02-21 17:59:02,275] {models.py:1407} INFO - 
--------------------------------------------------------------------------------
Starting attempt 1 of 2
--------------------------------------------------------------------------------

[2018-02-21 17:59:02,281] {models.py:1428} INFO - Executing <Task(BashOperator): run_new_listings_from_remax> on 2018-02-19 02:10:00
[2018-02-21 17:59:02,281] {base_task_runner.py:115} INFO - Running: ['bash', '-c', 'airflow run property_data run_new_listings_from_remax 2018-02-19T02:10:00 --job_id 44 --raw -sd DAGS_FOLDER/property_data.py']
[2018-02-21 17:59:02,864] {base_task_runner.py:98} INFO - Subtask: [2018-02-21 17:59:02,863] {__init__.py:45} INFO - Using executor SequentialExecutor
[2018-02-21 17:59:02,944] {base_task_runner.py:98} INFO - Subtask: [2018-02-21 17:59:02,943] {models.py:189} INFO - Filling up the DagBag from /Users/sanjeevsreetharan/Documents/ribbon/airflow/dags/property_data.py
[2018-02-21 17:59:03,012] {cli.py:374} INFO - Running on host Sanjeevs-MacBook-Pro.local
[2018-02-21 17:59:03,024] {base_task_runner.py:98} INFO - Subtask: [2018-02-21 17:59:03,024] {bash_operator.py:70} INFO - Tmp dir root location: 
[2018-02-21 17:59:03,025] {base_task_runner.py:98} INFO - Subtask:  /var/folders/j3/554c399s6t9bxlwbvc0628dw0000gn/T
[2018-02-21 17:59:03,025] {base_task_runner.py:98} INFO - Subtask: [2018-02-21 17:59:03,025] {bash_operator.py:80} INFO - Temporary script location: /var/folders/j3/554c399s6t9bxlwbvc0628dw0000gn/T/airflowtmpx1gco6mf//var/folders/j3/554c399s6t9bxlwbvc0628dw0000gn/T/airflowtmpx1gco6mf/run_new_listings_from_remaxod0e9tnn
[2018-02-21 17:59:03,026] {base_task_runner.py:98} INFO - Subtask: [2018-02-21 17:59:03,025] {bash_operator.py:88} INFO - Running command: python /jobs/property_data/run_remax_new_listings.py --city charlotte --state nc
[2018-02-21 17:59:03,031] {base_task_runner.py:98} INFO - Subtask: [2018-02-21 17:59:03,030] {bash_operator.py:97} INFO - Output:
[2018-02-21 17:59:03,080] {base_task_runner.py:98} INFO - Subtask: [2018-02-21 17:59:03,080] {bash_operator.py:101} INFO - python: can't open file '/jobs/property_data/run_remax_new_listings.py': [Errno 2] No such file or directory
[2018-02-21 17:59:03,082] {base_task_runner.py:98} INFO - Subtask: [2018-02-21 17:59:03,081] {bash_operator.py:105} INFO - Command exited with return code 2
[2018-02-21 17:59:03,088] {models.py:1595} ERROR - Bash command failed
Traceback (most recent call last):
  File "/Users/sanjeevsreetharan/anaconda3/lib/python3.6/site-packages/airflow/models.py", line 1493, in _run_raw_task
    result = task_copy.execute(context=context)
  File "/Users/sanjeevsreetharan/anaconda3/lib/python3.6/site-packages/airflow/operators/bash_operator.py", line 109, in execute
    raise AirflowException("Bash command failed")
airflow.exceptions.AirflowException: Bash command failed
[2018-02-21 17:59:03,090] {models.py:1616} INFO - Marking task as UP_FOR_RETRY
[2018-02-21 17:59:03,098] {models.py:1644} ERROR - Bash command failed
[2018-02-21 17:59:03,099] {base_task_runner.py:98} INFO - Subtask: /Users/sanjeevsreetharan/anaconda3/lib/python3.6/site-packages/airflow/utils/helpers.py:351: DeprecationWarning: Importing BashOperator directly from <module 'airflow.operators' from '/Users/sanjeevsreetharan/anaconda3/lib/python3.6/site-packages/airflow/operators/__init__.py'> has been deprecated. Please import from '<module 'airflow.operators' from '/Users/sanjeevsreetharan/anaconda3/lib/python3.6/site-packages/airflow/operators/__init__.py'>.[operator_module]' instead. Support for direct imports will be dropped entirely in Airflow 2.0.
[2018-02-21 17:59:03,099] {base_task_runner.py:98} INFO - Subtask:   DeprecationWarning)
[2018-02-21 17:59:03,099] {base_task_runner.py:98} INFO - Subtask: /Users/sanjeevsreetharan/anaconda3/lib/python3.6/site-packages/airflow/utils/helpers.py:351: DeprecationWarning: Importing PythonOperator directly from <module 'airflow.operators' from '/Users/sanjeevsreetharan/anaconda3/lib/python3.6/site-packages/airflow/operators/__init__.py'> has been deprecated. Please import from '<module 'airflow.operators' from '/Users/sanjeevsreetharan/anaconda3/lib/python3.6/site-packages/airflow/operators/__init__.py'>.[operator_module]' instead. Support for direct imports will be dropped entirely in Airflow 2.0.
[2018-02-21 17:59:03,099] {base_task_runner.py:98} INFO - Subtask:   DeprecationWarning)
[2018-02-21 17:59:03,100] {base_task_runner.py:98} INFO - Subtask: Traceback (most recent call last):
[2018-02-21 17:59:03,100] {base_task_runner.py:98} INFO - Subtask:   File "/Users/sanjeevsreetharan/anaconda3/bin/airflow", line 27, in <module>
[2018-02-21 17:59:03,100] {base_task_runner.py:98} INFO - Subtask:     args.func(args)
[2018-02-21 17:59:03,101] {base_task_runner.py:98} INFO - Subtask:   File "/Users/sanjeevsreetharan/anaconda3/lib/python3.6/site-packages/airflow/bin/cli.py", line 392, in run
[2018-02-21 17:59:03,101] {base_task_runner.py:98} INFO - Subtask:     pool=args.pool,
[2018-02-21 17:59:03,101] {base_task_runner.py:98} INFO - Subtask:   File "/Users/sanjeevsreetharan/anaconda3/lib/python3.6/site-packages/airflow/utils/db.py", line 50, in wrapper
[2018-02-21 17:59:03,101] {base_task_runner.py:98} INFO - Subtask:     result = func(*args, **kwargs)
[2018-02-21 17:59:03,101] {base_task_runner.py:98} INFO - Subtask:   File "/Users/sanjeevsreetharan/anaconda3/lib/python3.6/site-packages/airflow/models.py", line 1493, in _run_raw_task
[2018-02-21 17:59:03,101] {base_task_runner.py:98} INFO - Subtask:     result = task_copy.execute(context=context)
[2018-02-21 17:59:03,101] {base_task_runner.py:98} INFO - Subtask:   File "/Users/sanjeevsreetharan/anaconda3/lib/python3.6/site-packages/airflow/operators/bash_operator.py", line 109, in execute
[2018-02-21 17:59:03,101] {base_task_runner.py:98} INFO - Subtask:     raise AirflowException("Bash command failed")
[2018-02-21 17:59:03,101] {base_task_runner.py:98} INFO - Subtask: airflow.exceptions.AirflowException: Bash command failed
